deps/CVC4/base/check.h

```diff
@@ -206,7 +206,7 @@ class AssertArgumentException : public Exception
 
 #define Unimplemented() CVC4_FATAL() << "Unimplemented code encountered"
 
-#define InternalError() CVC4_FATAL() << "Internal error detected"
+//#define InternalError() CVC4_FATAL() << "Internal error detected"
```

Comment:
Can you add a one line comment (both here and in the output.h) just saying that it was excluded because of collisions when using ... compiler/setup?